### PR TITLE
Match Puma 3 startup messages [ci skip]

### PR DESCRIPTION
**Pull request by rubys**
Puma 3 startup messages are slightly different than the ones Puma 2 put out

### CODE REVIEW

File: guides/source/command_line.md [hunk 1]
```diff
@@ -65,11 +65,12 @@ $ bin/rails server
 => Booting Puma
 => Rails 5.0.0 application starting in development on http://0.0.0.0:3000
 => Run `rails server -h` for more startup options
-=> Ctrl-C to shutdown server
-Puma 2.15.3 starting...
-* Min threads: 0, max threads: 16
+Puma starting in single mode...
+* Version 3.0.2 (ruby 2.3.0-p0), codename: Plethora of Penguin Pinatas
+* Min threads: 5, max threads: 5
```
Comment (maclover7) [inline guides/source/command_line.md]: Is this the Rails default? Not sure if you did any tuning :)

File: guides/source/debugging_rails_applications.md [hunk 2]
```diff
@@ -314,11 +314,12 @@ For example:
 => Booting Puma
 => Rails 5.0.0 application starting in development on http://0.0.0.0:3000
 => Run `rails server -h` for more startup options
-=> Ctrl-C to shutdown server
-Puma 2.15.3 starting...
-* Min threads: 0, max threads: 16
+Puma starting in single mode...
+* Version 3.0.2 (ruby 2.3.0-p0), codename: Plethora of Penguin Pinatas
+* Min threads: 5, max threads: 5
```
Comment (maclover7) [inline guides/source/debugging_rails_applications.md]: ditto

**Comment by maclover7** [comment]
One small comment, otherwise LGTM. :)

cc @schneems

File: guides/source/command_line.md (hunk 1 above)
Comment (rubys) [inline guides/source/command_line.md]: I believe so, but feel free to start a server yourself and compare. :)

I do, however, know that the "=> Ctrl-C to shutdown server" previously emitted by Rails is gone.

**Pull request merged**